packages/core/src/components/card/card.scss

```diff
@@ -32,6 +33,14 @@
     background-color: var(--ix-card-background, transparent);
     border-top-left-radius: var(--theme-default-border-radius);
     border-top-right-radius: var(--theme-default-border-radius);
+    overflow: hidden;
+  }
+
+  .card-content-wrapper {
+    padding: 1rem;
```

Comment:
In case ix-card-content was is not used this will create padding that was not there before.
Resulting VRT snapshot:

<img width="1141" alt="Screenshot 2025-01-13 at 15 47 54" src="https://github.com/user-attachments/assets/fdc9c4aa-5c64-4b62-90a5-94b3e540008d" />